sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -106,11 +106,11 @@ def __init__(
         if 'message_encode_policy' in kwargs:
             self._config.message_encode_policy = kwargs['message_encode_policy'] or NoEncodePolicy()
         else:
-            self._config.message_encode_policy = TextXMLEncodePolicy()
+            self._config.message_encode_policy = NoEncodePolicy()
```

Comment:
Could be simplified to 
```python
self._config.message_encode_policy = kwargs.get('message_encode_policy', NoEncodePolicy())
```

Reply:
hmm, None and NoEncodepolicy are not same.

Reply:
@rakshith91, what is the semantic difference (for this method) between explicitly passing in `None` as the `message_encode_policy` and using `NoEncodePolicy`?

Reply:
We don't expose NoEncodePolicy() . Not passing anything or passing None explicitly will both  have NoEncodePolicy

Reply:
https://github.com/rakshith91/azure-sdk-for-python/blob/text_encoding_queue/sdk/storage/azure-storage-queue/tests/test_queue_encodings.py#L196

https://github.com/rakshith91/azure-sdk-for-python/blob/text_encoding_queue/sdk/storage/azure-storage-queue/tests/test_queue_encodings.py#L65

These two tests will explain it better...the first one has a None passed..the second has nothing...it's still NoEncodePolicy in both cases.

Reply:
With the line I suggest it's still NoEncodePolicy in both case. I don't understand your point? Both are equivalent, but one is taking 4 lines :)